ush/rocoto/setup_workflow.py

```diff
@@ -42,7 +42,7 @@ def main():
         print(f'input arg:     --expdir = {repr(args.expdir)}')
         sys.exit(1)
 
-    gfs_steps = ['prep', 'anal', 'analdiag', 'analcalc', 'gldas', 'fcst', 'postsnd', 'post', 'vrfy', 'arch']
+    gfs_steps = ['prep', 'analcalc', 'gldas', 'fcst', 'postsnd', 'postanl', 'post', 'vrfy', 'arch']
```

Comment:
I will have to understand the changes in this script with the changes in the PR that consolidates the XML generation.  I can give feedback on this part separately.

Reply:
Agreed.  Your refactoring of the XML generation take priority.  I can work with you add JEDI pieces into the refactored XML setup scripts.